README.md

```diff
@@ -722,6 +722,15 @@ Each operation will return the following values:
 3. `err` - An error object.  This will be nil if the operation was successful, or non-nil  
 if an error occurred.
 
+Here is an example of how to access the response and get additional Information
+beyond the response object:
+
+```go
+result, detailedResponse, err := myService.GetResource(options)
+
+responseHeaders := detailedResponse.GetHeaders()
```

Comment:
add an example of retrieving a specific header from `responseHeaders` (use same header name in each language's example).